### PR TITLE
Save SQLSmith results to the CI DB

### PR DESCRIPTION
**Pull request by akuzm**
Mostly we are interested in stacktraces and failed queries here.

### CODE REVIEW

File: scripts/upload_ci_stats.sh [hunk 1]
```diff
@@ -17,8 +10,10 @@ fi
 
 PSQL=(psql "${CI_STATS_DB}" -qtAX "--set=ON_ERROR_STOP=1")
```
Comment (mkindahl) [inline scripts/upload_ci_stats.sh:11]: This looks like a typo, but not sure.

**Review by mkindahl — APPROVED** [review]
I see no obvious problems with the shell script, but it is quite complicated and use a lot of "interesting" features of bash, so it might be better to re-write this in Python for maintainability.

File: scripts/upload_ci_stats.sh (hunk 1 above)
Comment (akuzm) [reply scripts/upload_ci_stats.sh:11]: Which part exactly? Looks correct to me...

Comment (mkindahl) [reply scripts/upload_ci_stats.sh:11]: I see below that it is creating a list and then expanding it, so it seems correct, although it is one of the "interesting" features I mention that is probably hard to maintain. First, it seemed to be missing a `$`, but that would not work either.

Comment (akuzm) [reply scripts/upload_ci_stats.sh:11]: Well, that's the usual bash thing, using arrays to save a command line. It's normally done this way not to mangle the arguments that have spaces in them. For this case it doesn't matter I guess. The `$` -- you mean before `ON_ERROR_STOP` or where? `ON_ERROR_STOP` is a psql internal variable, I'm setting it to true so that it fails on errors inside an SQL script.

People are always telling me that about bash :) I guess I'll try to write the next script in Python. It's inconvenient when you need to just run some programs and capture output, but it has much less weird stuff, I agree.

Comment (mkindahl) [reply scripts/upload_ci_stats.sh:11]: > Well, that's the usual bash thing, using arrays to save a command line. It's normally done this way not to mangle the arguments that have spaces in them. For this case it doesn't matter I guess. The `$` -- you mean before `ON_ERROR_STOP` or where? `ON_ERROR_STOP` is a psql internal variable, I'm setting it to true so that it fails on errors inside an SQL script.

Yeah, that is what I realized.

> People are always telling me that about bash :) I guess I'll try to write the next script in Python. It's inconvenient when you need to just run some programs and capture output, but it has much less weird stuff, I agree.

Not important at this point, but yes, bash has a lot of weird stuff that can be hard to follow. :)

**Review by fabriziomello — APPROVED** [review]
